doc/ref_model/chapters/chapter01.md

```diff
@@ -31,14 +31,14 @@ Initially organized early in 2019, the Common Network Function Virtualisation In
 
-To put this effort in perspective, over the past few years, the telecom industry has been going through a massive technology revolution by embracing software defined networking and cloud architecture principles, in pursuit of the goal of achieving more flexibility, agility and operational efficiency. At a high level, the main objective of NFV (Network Function Virtualization) is the ability to use general purpose standard COTS (Commercial off the Shelf) compute, memory and storage hardware platforms to run multiple Virtualised Network Functions.  VNFs is the general term that covers any type of virtualized application wiether it be in the form of a Virtual Machine (VM) or a containerized application.  Earlier common infrastructure models built on the previous assumption that networking applications are typically built on discrete hardware, do not offer the level of flexibility and agility needed for the support of newer networking technologies such as 5G, intelligent networks and Edge computing.  By running network applications as software rather than on purpose-built hardware, as it has been done since the early 1990’s, the operators aspire to realize operational efficiencies, and capital expense savings.  These Software Defined Network (SDN) applications are increasingly being used by telecom operators to support their internal and customer facing network infrastructures.  The need for a common model across the industry to facilitate more rapid adoption is clear.
+To put this effort in perspective, over the past few years, the telecom industry has been going through a massive technology revolution by embracing software defined networking and cloud architecture principles, in pursuit of the goal of achieving more flexibility, agility and operational efficiency. At a high level, the main objective of NFV (Network Function Virtualization) is the ability to use general purpose standard COTS (Commercial off the Shelf) compute, memory and storage hardware platforms to run multiple Virtualised Network Functions.  VNFs is the general term that covers any type of virtualized application whether it be in the form of a Virtual Machine (VM) or a containerized application.  Earlier common infrastructure models built on the previous assumption that networking applications are typically built on discrete hardware, do not offer the level of flexibility and agility needed for the support of newer networking technologies such as 5G, intelligent networks and Edge computing.  By running network applications as software rather than on purpose-built hardware, as it has been done since the early 1990’s, the operators aspire to realize operational efficiencies, and capital expense savings.  These Software Defined Network (SDN) applications are increasingly being used by telecom operators to support their internal and customer facing network infrastructures.  The need for a common model across the industry to facilitate more rapid adoption is clear.
```

Comment:
This paragraph is verbose and there are issues with certain statements. For example,

"By running network applications as software **rather** than on purpose-built hardware, as it has been done since the early 1990’s, the operators aspire to realize operational efficiencies, and capital expense savings. " Emphasis mine to indicate that what is the software running on rather than "purpose-built hardware." Also, aren't all applications software? "as it has been done since the early 1990’s" -- network software on purpose-build hardware has existed at least since the 1970's but more importantly not sure what the value add is.

First Crack at simplification:

The main objective of NFV (Network Function Virtualization) is to enable Network Functions to be hosted on commodity rather than purpose-built hardware. Virtualised Network Functions (VNFs) allows multiple network functions to share hardware resources.  VNFs is used here as a generic term to refer to network functions that are designed to utilise Virtual Machines (VMs) or are designed as a cloud native function; where distinction is needed the latter will be referred to as CNFs. NFV leads to greater flexibility, agility, operational efficiency, and capital expense reductions. In the absence of specifications of the underlying infrastructure, VNF developers, as stated previously, create their own infrastructure requirements, specifications, and configurations. This leads to incompatible NFV infrastructure (NFVI) requirements between VNFs from different developers but also form the same developers. This proliferation of infrastructure needs doesn't allow the benefits of NFV to be fully realised and defines the need for an industry-wide ("telco") common NFVI specifications.

Reply:
@pgoyal01 I think some of the points you make in this paragraph are made in the paragraph that precedes it.  I see where you're going though, so can I offer this to replace the paragraph in question (recalling the end of the previous paragraph)

This incompatibility creates costs and delays for multiple parties in the NFVI ecosystem.  It is this incompatibility that the CNTT aims to address and, in so doing, help deliver on Network Function Virtualization's original promise of lower deployment costs, lower operational costs and higher agility.

Reply:
I think this has nothing to do with the changes we introduced (changing "wiether" to "wether"). If you would like to change other parts of the text please create an issue and a pr.